### PR TITLE
Gradle file refactoring

### PR DESCRIPTION
**Pull request by samtstern**
Addresses #131 

@claywilkinson please review.
@dileeppg does this solution work?

### CODE REVIEW

File: BasicSamples/TypeANumber/src/main/res/values/ids.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
       http://www.apache.org/licenses/LICENSE-2.0
 
-   Unless required by applicable law or agreed to in writing, software
+   Unless required byhttps://www.google.com/about/careers/ applicable law or agreed to in writing, software
```
Comment (claywilkinson) [inline BasicSamples/TypeANumber/src/main/res/values/ids.xml]: i think this is a mistake

File: BasicSamples/build.gradle [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 buildscript {
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.1.0'
+        classpath 'com.android.tools.build:gradle:1.0.0'
```
Comment (claywilkinson) [inline BasicSamples/build.gradle]: why go backwards?

File: BasicSamples/TrivialQuest2/build.gradle [hunk 3]
```diff
@@ -1,4 +1,4 @@
-apply plugin: 'android'
+apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion android_compile_version
```
Comment (claywilkinson) [inline BasicSamples/TrivialQuest2/build.gradle]: should this be changed to compileSdkVersion ${android_compile_version}

**Comment by samtstern** [comment]
Nope, only need the ${...} for inserting within a string.

File: BasicSamples/TrivialQuest2/build.gradle (hunk 3 above)
Comment (samtstern) [inline BasicSamples/TrivialQuest2/build.gradle]: Nope, only need the ${...} for inserting within a string.

**Comment by samtstern** [comment]
@claywilkinson comments addressed.

**Comment by dileeppg** [comment]
@samtstern @claywilkinson

I just downloaded the latest zip and imported the BaseGameUtils module into a new project. But Gradle Project Sync failed.

Error:(15, 0) No such property: android_compile_version for class: org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler_Decorated

**Pull request merged**